src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -14415,6 +14415,14 @@ static void M()
                 symbolValidator: module =>
                 {
                     AssertEx.Equal(new[] { "<>y__InlineArray2", "<>y__InlineArray3" }, getInlineArrayTypeNames(module));
+
+                    foreach (var inlineArrayType in getInlineArrayTypes(module))
+                    {
+                        Assert.Equal(System.Runtime.InteropServices.LayoutKind.Sequential, inlineArrayType.Layout.Kind);
+                        Assert.Single(inlineArrayType.GetAttributes(), isCompilerGeneratedAttribute);
```

Comment:
Also please add a dedicated test that verifies [what attributes were actually emitted](https://github.com/dotnet/roslyn/blob/935befe3518e902e58508e591a497c1ed7cbae4b/src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs#L12851-L12858)

Reply:
Instead of the current way of testing the PE symbols? Is it possible for the results to differ?